From 95d14416474848e4064a171923f99905e0091ac7 Mon Sep 17 00:00:00 2001
From: Alex Stewart <alex.stewart@ni.com>
Date: Tue, 12 Jul 2022 04:50:22 -0500
Subject: [PATCH] Makefile: set release information for v3.0.5

The release variables were not set for the original v3.0.5 release tag.
So `dkms --version` would erroneously report `dkms-3.0.4`.

Set the correct release information.

Signed-off-by: Alex Stewart <alex.stewart@ni.com>

Upstream-Status: Submitted [https://github.com/dell/dkms/pull/233]

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index d4735d4..132cb69 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
-RELEASE_DATE := "18 June 2022"
+RELEASE_DATE := "30 June 2022"
 RELEASE_MAJOR := 3
 RELEASE_MINOR := 0
-RELEASE_MICRO := 4
+RELEASE_MICRO := 5
 RELEASE_NAME := dkms
 RELEASE_VERSION := $(RELEASE_MAJOR).$(RELEASE_MINOR).$(RELEASE_MICRO)
 RELEASE_STRING := $(RELEASE_NAME)-$(RELEASE_VERSION)
